iron-overlay-backdrop.html

```diff
@@ -118,10 +118,14 @@
       this.style.zIndex = this._manager.backdropZ();
       // close only if no element with backdrop is left
       if (this._manager.getBackdrops().length === 0) {
+        // Read style before setting opened.
+        var cs = getComputedStyle(this);
+        var noAnimation = (cs.transitionDuration === '0s' || cs.opacity == 0);
```

Comment:
What is the extra condition about `opacity`? "Don't transition transparent things?"